public/js/clients/firefox.js

```diff
@@ -94,6 +96,11 @@ let APIClient = {
 
   navigate(url) {
     return currentTabTarget.activeTab.navigateTo(url);
+  },
+
+  getProperties(grip) {
```

Comment:
True, I'll change it, but we'll probably want to rethink it again later because there are a few other methods for querying objects